code/modules/mob/living/carbon/give.dm

```diff
@@ -1,52 +1,164 @@
-/mob/living/carbon/verb/give(mob/living/carbon/target in oview(1))
-	set category = "IC"
-	set name = "Give"
+/**
+ * Toggles the [/datum/click_intercept/give] on or off for the src mob.
+ */
+/mob/living/carbon/proc/toggle_give()
+	if(HAS_TRAIT(src, TRAIT_OFFERING_ITEM))
+		to_chat(src, "<span class='warning'>You're already offering an item to someone!</span>")
```

Comment:
why can't you offer same item to multiple people tho? Would be both fun and practical, wouldn't it?
(although the code would be messy, i guess)

Reply:
It could be done, but yeah, I think the code would be a bit messy. I don't imagine too many situations where you would want to offer your item to multiple people at once. Usually you want to hand a singular person an item.